### DISCUSSION TITLE
无法正常运行

### DISCUSSION THREAD

**Discussion by SunYong0821** [zai-org/ChatGLM3#698] [General]
![image](https://github.com/THUDM/ChatGLM3/assets/7601063/a6db81b6-1c57-4108-8ec5-ac8a671bb16f)
![image](https://github.com/THUDM/ChatGLM3/assets/7601063/184dcd31-6220-4d7d-a19e-76a3f2f18f8b)

ubuntu  A100 完全按照git来部署的。。。

**Comment by zRzRzRzRzRzRzR** [comment]
模型没更新，依赖没对或者出现了非法字符，字典越界了估计。尝试更新最新的hf代码

**Reply by SunYong0821** [reply]
这是我今天刚github下载的模型。。。。。。。难道hf和github上的不一样吗
